csv2json/csv2json.py

```diff
@@ -0,0 +1,58 @@
+import  json
+import csv
+
+
+def readcsv(fname:str) -> (list):

```

Comment:
The parentheses in `(list)` is redundant. Using `list` is OK. Moreover, Using `List[str]` from typing module is better.
```suggestion
from typing import List
#...
def readcsv(file_name:str) -> List[str]:
```

Reply:
I'd suggest using `read_csv` insetad of `readcsv`.  Also an additional space between `file_name` and `str` would be better (i.e., `file_name: str`)